README.md

```diff
@@ -8,154 +8,66 @@ The ArkCase architecture is described here: https://www.arkcase.com/developer-su
-      <UpgradeProtocol className="org.apache.coyote.http2.Http2Protocol" />
-      <SSLHostConfig protocols="TLSv1.2" certificateVerification="none">
-        <Certificate certificateFile="${user.home}/.arkcase/acm/private/acm-arkcase.crt"
-                    certificateKeyFile="${user.home}/.arkcase/acm/private/acm-arkcase.rsa.pem"
-                    certificateChainFile="${user.home}/.arkcase/acm/private/arkcase-ca.crt"
-                    type="RSA" />
-      </SSLHostConfig>
-    </Connector>
+To install ArkCase using helm chart we need to add the helm chart to our local repository.
+* Run the following command to add arkcase to repository and to update it. 
+```bash
+helm repo add arkcase https://arkcase.github.io/ark_helm_charts/
+helm repo update
 ```
-
-Also, search for the text `Listener className="org.apache.catalina.core.AprLifecycleListener"`, and make sure to add the `useAprConnector="true"` attribute, so it ends like this:
-
-```xml
-<Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="on" useAprConnector="true"/>
-``` 
-
-#### Tomcat setenv.sh file
-
-Create the file `bin/setenv.sh`, mark it executable, and set the contents as the following, *being careful to set the correct path to the Tomcat native library*:
-
+* Create the namespace to deploy arkcase
 ```bash
-#!/bin/sh
-
-### MacOS X note: replace {user.home} with the actual path to your home folder, e.g. /Users/dmiller
-export JAVA_OPTS="-Djava.net.preferIPv4Stack=true -Duser.timezone=GMT  -Djavax.net.ssl.keyStorePassword=password -Djavax.net.ssl.trustStorePassword=password -Djavax.net.ssl.keyStore=${user.home}/.arkcase/acm/private/arkcase.ks -Djavax.net.ssl.trustStore=${user.home}/.arkcase/acm/private/arkcase.ts -Dspring.profiles.active=ldap -Dacm.configurationserver.propertyfile="${user.home}/.arkcase/acm/conf.yml -Xms1024M -Xmx1024M"
-
-export NODE_ENV=development
-
-export CATALINA_OPTS="$CATALINA_OPTS -Djava.library.path=(PATH TO THE TOMCAT NATIVE LIBRARY)
-# MacOS Example: export CATALINA_OPTS=/usr/local/opt/tomcat-native/lib"
-
-export CATALINA_PID=$CATALINA_HOME/temp/catalina.pid
+kubectl create namespace arkcase
+```
+* Now that the repository is installed and updated we can proceed to deploy the ArkCase CE.
+```bash
+helm install arkcase arkcase/app -n arkcase
+```
+* Watch Pods Until Running.
+```bash
+watch kubectl get pods -n arkcase
 ```
 
-On MacOS X, you have to replace `file:${user.home}` in the above script, with the actual full path to your home folder.
-
-#### Start Tomcat
-
-Now you should be able to start Tomcat: `$TOMCAT_HOME/bin/startup.sh`.  
+### Access ArkCase from Browser
 
-To shutdown Tomcat: `$TOMCAT_HOME/bin/shutdown.sh -force`.
+To access ArkCase from a browser, we need to port forward the service. Follow these steps to achieve that:
 
-### Deploy the ArkCase war file
+1. **Get the Cluster IP of the `core` service:**
+   - Use this command to get the Cluster IP: `kubectl get service core` (Note the CLUSTER-IP).
```

Comment:
You might want to add the `-n arkcase` to all these kubectl commands, since you used that namespace in the install step.

Reply:
Will update Dave